index.js

```diff
@@ -14,11 +14,17 @@
 //. [![CircleCI](https://img.shields.io/circleci/project/github/sanctuary-js/sanctuary/master.svg)](https://circleci.com/gh/sanctuary-js/sanctuary/tree/master)
 //. [![Gitter](https://img.shields.io/gitter/room/badges/shields.svg)](https://gitter.im/sanctuary-js/sanctuary)
 //.
-//. Sanctuary is a functional programming library inspired by Haskell
-//. and PureScript. It's stricter and more opinionated than [Ramda][],
-//. and provides a similar suite of functions.
+//. Sanctuary is a functional programming library inspired by Haskell and
+//. PureScript. It provides a similar suite of functions as [Ramda][],
+//. but in a stricter and more opinionated way, ensuring errors are
+//. properly handled.
+//. Sanctuary promotes programs composed of simple, pure functions.
+//. Such programs are easy to test, comprehend, and maintain, and a
+//. pleasure to write.
+//. Sanctuary provides two [Fantasy Land][]-compatible data types,
+//. Maybe and Either, which mean that even functions which may fail
+//. — such as `head` — are composable.
```

Comment:
@davidchambers yes, do you want me to link them directly (e.g. [Maybe](https://sanctuary.js.org/#maybe-type))?